README.md

```diff
@@ -8,13 +8,22 @@ It can also, optionally, populate this stack with real data if local development
 How to use it
 -------------
 
-On MacOS, run
+The following command will build and execute the inframock environment loading all the experiment data found in `./data` folder:
 
-    docker-compose up --build
+    make run
 
-On Linux, run
+**Note** On Linux you will need to set first the environment variable `DOCKER_COMPOSE_FILE` to point to the linux docker-compose file:
+
+    export DOCKER_COMPOSE_FILE='docker-compose.linux-dev.yaml'
```

Comment:
This may fall outside of the scope of this PR, but I think it is worth being considered here.

I have not tested it with these specific changes, but `docker-compose.linux-dev.yaml` [can be limited to have only overrides](https://docs.docker.com/compose/extends/#multiple-compose-files). According to `git diff --no-index docker-compose.yaml docker-compose.linux-dev.yaml`, that would be 

```
version: "3.2"
services:
  localstack:
    extra_hosts:
      - "host.docker.internal:host-gateway"
```

Then, for Linux, `docker-compose` could be invoked 

```
> docker-compose -f docker-compose.yaml -f docker-compose.linux-dev.yaml ...
```

while for other operating systems, the second `docler-compose` file would not be used.

**Warning:** `extra_hosts` is not explicitely mentioned in the documentation for how overrides are processed, so this approach should be tested before any changes to the docs or the `makefile`.

Reply:
This would be nice to avoid having to manage the same changes in two docker-compose files at the same time.

Reply:
@ogibson-biomage code now uses the overrides, could you test if it does work in Linux?